packages/forms/src/Components/Repeater.php

```diff
@@ -32,6 +32,8 @@ class Repeater extends Field implements Contracts\CanConcealComponents
 
     protected bool | Closure $isReorderable = true;
 
+    protected int | Closure $sortableAnimation = 300;
```

Comment:
Naming should probably be changed to be component-specific like `reorderAnimationDuration` or similar. Should be added to all places where sortable is used. Let's wait on @danharrin's opinion before spending more time on this.

Reply:
`reorderAnimationDuration` sounds good

Reply:
Cool, could you implement this for all places where sortable is used, @ijpatricio?

Reply:
Yeps, will do!